ironic/drivers/modules/drac/raid.py

```diff
@@ -202,6 +228,59 @@ def delete_virtual_disk(node, virtual_disk):
         raise exception.DracOperationError(error=exc)
 
 
+def reset_raid_config(node, raid_controller):
+    """Delete all virtual disk and unassign all hotspares physical disk
+
+    :param node: an ironic node object.
+    :param raid_controller: id of the RAID controller.
+    :returns: a dictionary containing the is_commit_required needed key with a
+              boolean value indicating whether a config job must be created
+              for the values to be applied.
+    :raises: DracOperationError on an error from python-dracclient.
+    """
+    drac_job.validate_job_queue(node)
+
+    client = drac_common.get_drac_client(node)
+
+    try:
+        return client.reset_raid_config(raid_controller)
+    except drac_exceptions.BaseClientException as exc:
+        LOG.error('DRAC driver failed to delete all virtual disk '
+                  'and unassign all hotspares '
+                  'for node %(node_uuid)s. '
+                  'Reason: %(error)s.',
+                  {'raid_controller_fqdd': raid_controller,
+                   'node_uuid': node.uuid,
+                   'error': exc})
+        raise exception.DracOperationError(error=exc)
+
+
+def clear_foreign_config(node, raid_controller):
+    """Free up the foreign drives.
+
+    :param node: an ironic node object.
+    :param raid_controller: id of the RAID controller.
+    :returns: a dictionary containing the is_commit_required needed key with a
```

Comment:
The dictionary will contain both is_commit_required and is_reboot_required. Can you update the comment to include both?

Reply:
Done